### DISCUSSION TITLE
Extension installation aborted.

### DISCUSSION THREAD

**Discussion by troy-cf** [joomengine/Joomla-Component-Builder#961] [General]
I tried to install JCB, I downloaded the zip file : pkg-component-builder-master and when I try to install JCB it fails with the following:

JInstaller: :Install: Error SQL Field 'addcontributors' doesn't have a default value
Extension installation aborted.
Package Install: There was an error installing an extension: joomla__Component-Builder__v3.1.28.zip

Error installing package

Is there something I need to do differently?

My Joomla version : Joomla! Version | Joomla! 4.2.9 Stable

**Comment by TLWebdesign** [comment]
Yes you need to install this in Joomla 3. not 4/5. JCB is not ready to be installed on J4/5. Only on J3. And then you can compile your components for J4/5.
